src/app/components/normal-component/count-down/count-down.component.ts

```diff
@@ -132,7 +121,7 @@ export class CountDownComponent implements OnInit, OnDestroy {
     public isStarted(): boolean {
         return this.started;
     }
-    public getTimeStyle(): { [key: string]: string } {
+    public getTimeClass(): string {
```

Comment:
Je suis en train de faire cette modif aussi dans ma branche, du coup, pour simplifier et safiser le conflit:

public getBackgroundColor(): { [key: string]: string } {
        const buttonStyle: { [key: string]: string } = this.getTimeClass();
        return { 'background-color': buttonStyle['background-color'] };
}

Cette fonction fournissait les style du bouton +.

Qu'en ferais-tu ?

Reply:
hmm partir simplement sur un is-danger pour la classe du bouton si danger il y a, sinon pas de classe particulière je dirais ?